clang-tools-extra/test/clang-doc/basic-project.test

```diff
@@ -54,306 +54,249 @@
-// HTML-SHAPE-NEXT: <script src="{{.*}}index.js"></script>
-// HTML-SHAPE-NEXT: <header id="project-title"></header>
-// HTML-SHAPE-NEXT: <main>
-// HTML-SHAPE-NEXT:   <div id="sidebar-left" path="GlobalNamespace" class="col-xs-6 col-sm-3 col-md-2 sidebar sidebar-offcanvas-left"></div>
-// HTML-SHAPE-NEXT:   <div id="main-content" class="col-xs-12 col-sm-9 col-md-8 main-content">
-// HTML-SHAPE-NEXT:     <h1>class Shape</h1>
-// HTML-SHAPE-NEXT:     <p>Defined at line 8 of file {{.*}}Shape.h</p>
-// HTML-SHAPE-NEXT:     <div>
-// HTML-SHAPE-NEXT:       <div>
-// HTML-SHAPE-NEXT:         <p> Provides a common interface for different types of shapes.</p>
-// HTML-SHAPE-NEXT:       </div>
-// HTML-SHAPE-NEXT:     </div>
-// HTML-SHAPE-NEXT:     <h2 id="Functions">Functions</h2>
-// HTML-SHAPE-NEXT:     <div>
-// HTML-SHAPE-NEXT:       <h3 id="{{([0-9A-F]{40})}}">~Shape</h3>
-// HTML-SHAPE-NEXT:       <p>public void ~Shape()</p>
-// HTML-SHAPE-NEXT:       <p>Defined at line 13 of file {{.*}}Shape.h</p>
-// HTML-SHAPE-NEXT:       <div>
-// HTML-SHAPE-NEXT:         <div></div>
-// HTML-SHAPE-NEXT:       </div>
-// HTML-SHAPE-NEXT:       <h3 id="{{([0-9A-F]{40})}}">area</h3>
-// HTML-SHAPE-NEXT:       <p>public double area()</p>
-// HTML-SHAPE-NEXT:       <div>
-// HTML-SHAPE-NEXT:         <div></div>
-// HTML-SHAPE-NEXT:       </div>
-// HTML-SHAPE-NEXT:       <h3 id="{{([0-9A-F]{40})}}">perimeter</h3>
-// HTML-SHAPE-NEXT:       <p>public double perimeter()</p>
-// HTML-SHAPE-NEXT:       <div>
-// HTML-SHAPE-NEXT:         <div></div>
-// HTML-SHAPE-NEXT:       </div>
-// HTML-SHAPE-NEXT:     </div>
-// HTML-SHAPE-NEXT:   </div>
-// HTML-SHAPE-NEXT:   <div id="sidebar-right" class="col-xs-6 col-sm-6 col-md-2 sidebar sidebar-offcanvas-right">
-// HTML-SHAPE-NEXT:     <ol>
-// HTML-SHAPE-NEXT:       <li>
-// HTML-SHAPE-NEXT:         <span>
-// HTML-SHAPE-NEXT:           <a href="#Functions">Functions</a>
-// HTML-SHAPE-NEXT:         </span>
-// HTML-SHAPE-NEXT:         <ul>
-// HTML-SHAPE-NEXT:           <li>
-// HTML-SHAPE-NEXT:             <span>
-// HTML-SHAPE-NEXT:               <a href="#{{([0-9A-F]{40})}}">~Shape</a>
-// HTML-SHAPE-NEXT:             </span>
-// HTML-SHAPE-NEXT:           </li>
-// HTML-SHAPE-NEXT:           <li>
-// HTML-SHAPE-NEXT:             <span>
-// HTML-SHAPE-NEXT:               <a href="#{{([0-9A-F]{40})}}">area</a>
-// HTML-SHAPE-NEXT:             </span>
-// HTML-SHAPE-NEXT:           </li>
-// HTML-SHAPE-NEXT:           <li>
-// HTML-SHAPE-NEXT:             <span>
-// HTML-SHAPE-NEXT:               <a href="#{{([0-9A-F]{40})}}">perimeter</a>
-// HTML-SHAPE-NEXT:             </span>
-// HTML-SHAPE-NEXT:           </li>
-// HTML-SHAPE-NEXT:         </ul>
-// HTML-SHAPE-NEXT:       </li>
-// HTML-SHAPE-NEXT:     </ol>
-// HTML-SHAPE-NEXT:   </div>
-// HTML-SHAPE-NEXT: </main>
+// HTML-SHAPE: <h1>class Shape</h1>
+// HTML-SHAPE: <p>Defined at line 8 of file {{.*}}Shape.h</p>
+// HTML-SHAPE: <div>
+// HTML-SHAPE:   <div>
```

Comment:
Do you need to match these? are they important? same w/ all the lines that only have tags...

Reply:
good point I remove everything except for relevant infromation